Shawntru/FitLit-refactor
# Fix/add conditionals

## Is this a fix or a feature?
Feature

## What is the change?
Conditionals have been added for the input fields for updating data.

## Where should the reviewer start?
src/scripts.js line 54
src/css/style.scss line 196
src/index.html  line 49

## How should this be tested?
As a user try and press any of the update data buttons.
An error alert should appear. Once the data is entered the button will then be clickable. 

@Shawntru 
@adavidson02
@sarah01123
